### DISCUSSION TITLE
CSS Variables for gradients

### DISCUSSION THREAD

**Discussion by bakastev** [webstudio-is/webstudio-community#197] [General]
I've tested some css variables and the most are working great. But setting up variables for gradients seems not working for the moment. I have have to put code manually, to set up gradients. Maybe i'm wrong...?!

**Comment by TrySound** [comment]
Please attach a screenshot what is not working for you

**Reply by bakastev** [reply]
I've variables for gradients  like var(--gradient-15) in the global root. If i try to set up this variable for a container/card under backgrounds, the gradient is not showing up. I still have to copy and paste the code attached to this variable (linear-gradient(to bottom right,#c7d2fe,#fecaca,#fef3c7). If i choose a variable with a normal bg color, the variable ist working. 

![Capture-2024-10-16-113938](https://github.com/user-attachments/assets/61b0c3ff-05e5-4d22-8539-887c69b1bc37)
![Capture-2024-10-16-113816](https://github.com/user-attachments/assets/074673ce-40f4-47dd-bb6c-7fe2c411b4e0)

**Reply by bakastev** [reply]
Setting the variable here is not working too. For the moment i don't see any possibility to use variables for gradients...
![image](https://github.com/user-attachments/assets/da59a3b3-ec07-4095-8b42-e1f0758b8a16)

**Reply by kof** [reply]
can you share a link with view permissions?

**Reply by bakastev** [reply]
Sure, but there is not that much to see and maybe not a local issue. I've just imported the Craft/Open Props Variables. @kof are you able to put just gradient variables inside background color? https://p-fcfb0365-74ec-4ff5-91bf-097d970b4519.apps.webstudio.is/?authToken=f36fcc7c-c4b9-4799-bf12-2ffc85d78d73&mode=preview

**Reply by TrySound** [reply]
Works for me
![image](https://github.com/user-attachments/assets/7115fe37-751a-4c5c-8890-799d1b9da8f5)
